test/framework/src/main/java/org/elasticsearch/index/mapper/MapperTestCase.java

```diff
@@ -441,6 +441,10 @@ protected String[] getParseMinimalWarnings() {
         return Strings.EMPTY_ARRAY;
     }
 
+    protected String[] getParseMinimalWarnings(IndexVersion indexVersion) {
```

Comment:
Added this method so we can specify warnings that will be included when using mappings that go back to specific index versions.